utils/update-consul-conf

```diff
@@ -227,7 +227,7 @@ for id in $IOS_IDs; do
     append_ios_svc $id
 done
 
-s3port=28081
+s3port=28071
```

Comment:
Going further, I was actually thinking instead of hardcoding the port here, can we take it as an argument? was thinking of having a hare configuration file (/etc/sysconfig/hare) in which we can specify all the Hare specific environmental variables, e.g. s3port. Or I think we should read the s3port somewhere from a s3server configuration file. Because everytime the port number changes we will have to make a code change and if we can achieve this without code change then it is better.
@knekrasov , what do you think?

Reply:
Although I'm not that sure that S3 port deserves an externalized configuration parameter, in general it would be nice to have some Hare config file with various parameters such as log level, log format, number of worker threads (#1612) etc.

So it is nice to have but it deserves a separate PR.

Reply:
Yeah, separate PR sounds good. Just that we need not have to change the code every time some port changes. These things need to be configurable. Let's create a ticket to identify all the configurable Hare parameters and make them configurable.
@SwapnilGaonkar7 Can you please create a ticket for the same?

Reply:
Done, https://jts.seagate.com/browse/EOS-21132